src/test/scala/kamon/http4s/HttpMetricsSpec.scala

```diff
@@ -70,22 +70,12 @@ class HttpMetricsSpec extends WordSpec
       val requests = List
         .fill(100) {
           get("/tracing/ok")(server, client)
-        }
-        .parSequence_
+        }.parSequence_
 
       val test = IO {
-        eventually(timeout(5 seconds)) {
```

Comment:
Due to the way the test is setup, `requests *> test` the test is only run after all requests have completed therefore there's no need to use eventually.